tests/bot/exts/backend/test_error_handler.py

```diff
@@ -47,7 +47,7 @@ async def test_error_handler_command_not_found_error_not_invoked_by_handler(self
         )
         self.cog.try_silence = AsyncMock()
         self.cog.try_get_tag = AsyncMock()
-        self.cog.try_run_eval = AsyncMock(return_value=False)
+        self.cog.try_run_fixed_codeblock = AsyncMock(return_value=False)
```

Comment:
`self.cog.try_run_fixed_codeblock` is not used in the test and even after removing it the tests run fine.
Although I do understand that it is used in `self.cog.on_command_error` but even after removing it the tests run fine, so is it really necessary to have this?

Reply:
I guess it still works because it would hit the line `if new_ctx.command is None:` which would probably be True. I think it makes sense to leave it though, since this test isn't meant to test that command so it makes the test more contained.

Another alternative would be to make it so `try_run_fixed_codeblock` is actually tested with both True and False, but I don't think it's worth it, since the test is already pretty complicated compared to the very simple code that it's actually testing.